juntagrico/dao/memberdao.py

```diff
@@ -45,6 +45,12 @@ def members_for_create_subscription():
     def members_for_email():
         return Member.objects.exclude(inactive=True)
 
+    @staticmethod
+    def get_jobs_for_current_day():
+        today = timezone.now()
+        daystart = datetime(year=today.year, month=today.month, day=today.day, hour=0, second=0)
```

Comment:
https://docs.python.org/3.7/library/datetime.html#datetime.date.today
https://docs.python.org/3.7/library/datetime.html#datetime.time
https://docs.python.org/3.7/library/datetime.html#datetime.datetime.combine

Reply:
changed to datetime.combine(date.today(), time.min)